.travis.yml

```diff
@@ -9,7 +9,7 @@ cache:
   - node_modules
   - server/node_modules
   - client/node_modules
-after_success: concurrently 'cd server/ && npm run coverage' 'cd client/ && npm run coverage'
+after_success: concurrently 'cd bartop-api/ && npm run coverage' 'cd bartop-ui/ && npm run coverage'
```

Comment:
Has the Travis build really been using concurrently this whole time? How is it getting that package if we uninstalled it? Should we take that out?

Reply:
Weird - yeah it definitely needs to be taken out, not sure how it wasn't breaking 🤔

Reply:
Yeah that's what I was thinking